config/default/ldap.yaml

```
@@ -17,3 +17,5 @@ service:
     - '52.167.253.43/32'      # Accept inbound LDAPS from public.aks.jenkins.io
     - '34.211.101.61/32'      # Accept inbound connections from Linux Foundation test machine
     - '44.240.22.235/32'      # Accept inbound connections from Linux Foundation prod machine
+    - '20.75.10.224/29'       # Accept inbound connections from publick8s public IPs
```

Are the ci.jenkins.io agents allocated from the publick82 cluster or are they on a different cluster?  Are they receiving a public IP address in this address range when they are allocated?  Would a malicious program on a ci.jenkins.io agent be able to connect to the LDAP server?
```suggestion
    - '20.75.10.224/29'       # Accept inbound connections from publick8s public IPs
```

---

No ci agents are not running on publick8s and should never be allowed